### PR TITLE
Translate terms in navigation

### PR DESCRIPTION
**Pull request by remibetin**

### CODE REVIEW

File: navigation.yml [hunk 1]
```diff
@@ -2,12 +2,14 @@
...
     es: "Fundamentos de accesibilidad"
     fr: "Fondamentaux de l'accessibilité"
+    pl: "Podstaw dostępności"
   translate: true
   pages:
   - name:
       en: 'Accessibility Fundamentals'
       es: 'Fundamentos de accesibilidad'
       fr: "Fondamentaux de l'accessibilité"
+      pl: "Podstaw dostępności"
```
Comment (zwiastunsw) [inline navigation.yml:12]:
```suggestion
      pl: "Podstawy dostępności"
```

File: navigation.yml [hunk 2]
```diff
@@ -741,7 +776,9 @@
       url: "/standards-guidelines/atag/no-code/"
     - name: "For Social Media Platforms"
       url: "/standards-guidelines/atag/social-media/"
-  - name: User Agents – UAAG
+  - name:
+      en: "User Agents – UAAG"
+      pl: "Programów użytkownika – UAAG"
```
Comment (zwiastunsw) [inline navigation.yml:781]:
```suggestion
      pl: "Programy użytkownika – UAAG"
```

**Review by zwiastunsw — APPROVED** [review]

File: navigation.yml [hunk 3]
```diff
@@ -578,18 +607,20 @@
...
-      en: 'Evaluation Tools'
+      en: "Evaluation Tools"
       fr: "Outils d'évaluation"
+      pl: "Narzędzi oceny"
     url: "/test-evaluate/tools/"
     translate: true
     pages:
-      - name: Selecting Evaluation Tools
+      - name:
+          en: "Selecting Evaluation Tools"
```
Comment (remibetin) [inline navigation.yml:617]: @zwiastunsw How would you translate the shortname of this resource into Polish?

Comment (zwiastunsw) [inline navigation.yml:617]:
```suggestion
          en: "Selecting Evaluation Tools"
          pl: "Wybór narzędzi oceny dostępności cyfrowej"
```

File: navigation.yml [hunk 4]
```diff
@@ -578,18 +607,20 @@
       - name: Required Fields
         url: "/test-evaluate/easy-checks/required-fields/"
   - name:
-      en: 'Evaluation Tools'
+      en: "Evaluation Tools"
       fr: "Outils d'évaluation"
+      pl: "Narzędzi oceny"
```
Comment (zwiastunsw) [inline navigation.yml:612]:
```suggestion
      pl: "Narzędzia oceny"
```

**Comment by zwiastunsw** [comment]
@remibetin : ready

**Comment by remibetin** [comment]
Thanks @zwiastunsw!

**Pull request merged**